api/konnect/v1alpha2/konnect_extension_types.go

```diff
@@ -224,6 +216,9 @@ const (
 	ClusterTypeK8sIngressController KonnectExtensionClusterType = "K8SIngressController"
 )
 
+// KonnectEndpoints is an alias for the Konnect endpoints v1alpha1.
+type KonnectEndpoints konnectv1alpha1.KonnectEndpoints
```

Comment:
This is not an alias but a separate type (which has the same layout as `konnectv1alpha1.KonnectEndpoints`).

I'd suggest that we define a separate type though, making these 2 completely independent and thus preventing any issues in the future if alpha1 type gets changed.

Reply:
Alright, here it is: fb1a24cc7c66f2cf93d1b2f0ced17ade7d7760ec